mathesar/exceptions/exceptions.py

```diff
@@ -0,0 +1,49 @@
+from collections import namedtuple
+from typing import List
+
+from django.utils.encoding import force_str
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+from mathesar.exceptions.error_codes import ErrorCodes
+
+ExceptionBody = namedtuple('ExceptionBody',
+                           [
+                               'code',
+                               'message',
+                               'field',
```

Comment:
Maybe name this `input_field` or `json_field`? Meaning would be a bit clearer, since SQLAlchemy uses the term `field` a lot (synonymous with our `column`).

Reply:
I was trying to make it work close with the drf tools, changing the name would lead to replicating a lot of existing code. I will see if I can change it to without breaking drf

Reply:
It should follow the [spec](https://wiki.mathesar.org/en/engineering/architecture/api-standards), which means the key should be `field`. API users should be unaware of SQLAlchemy and its nomenclature.